packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -250,12 +257,13 @@ QUnit.module('collapsible groups', moduleSetup, () => {
 
         const $headers = $element.find(toSelector(LIST_GROUP_HEADER_CLASS));
         const $items = $element.find(toSelector(LIST_ITEM_CLASS));
+        const secondGroupItemIsFocused = $items.eq(2).hasClass(FOCUSED_STATE_CLASS);
 
         $element.trigger('focusin');
 
         $headers.eq(1).trigger('dxclick');
         $headers.eq(1).trigger('dxclick');
-        assert.notOk($items.eq(2).hasClass('dx-state-focused'), 'first item of the second group is focused');
+        assert.notOk(secondGroupItemIsFocused, 'first item of the second group is focused');
```

Comment:
This test is evergreen now because u store `secondGroupItemIsFocused` before you focus items. Please do not create variables for this simple checks to avoid such errors.